python/lsst/ts/eas/eas_csc.py

```diff
@@ -99,7 +101,9 @@ async def configure(self, config):
 
     async def implement_simulation_mode(self, simulation_mode):
         if simulation_mode not in (0, 1):
-            raise salobj.ExpectedError(f"Simulation_mode={simulation_mode} must be 0 or 1")
+            raise salobj.ExpectedError(
```

Comment:
Please delete this method (implement_simulation_mode). Since you have defined valid_simulation_modes as a class variable it is no longer needed.